WMF Framework/EventLogging.xcdatamodeld/.xccurrentversion

```diff
@@ -1,8 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you remove this file entirely? It's not in the project anymore but I still see it in the WMF Framework directory.

It also may be worthwhile to add a migration step in `performUpdatesFromLibraryVersion` to delete the `sqlite` files in the app container.